samples/deployment/aws/build_ami_vftd.yaml

```diff
@@ -0,0 +1,196 @@
+# Create Basic VM For AMI BUILD
+- name: "Build AMI image for vFTD"
+  hosts: localhost
+  remote_user: ubuntu
+  gather_facts: no
+  vars:
+    ftd_build_id: "6.4.0-1304"
+    ec2_instance_type: "c4.xlarge"
+    ec2_base_image: "ami-038f5168"
+    : "ngfwv-stage-temp-kp.pem"
+  tasks:
+    - name: "Download QCOW2"
+      get_url:
+        url: "https://...path.../...to.../{{ ftd_build_id }}.qcow2"
+        dest: "./"
+
+    # Prepare Env Section
+    - name: "Create an AWS EC2 key pair"
+      ec2_key:
+        name: "{{ ansible_ssh_private_key_file }}"
```

Comment:
I guess, it should have `ansible_ssh_private_key_name` value as it attempts to create a key pair with a given name.

Reply:
And shouldn't these variables be defined in `vars` section?

Reply:
fixed